docs/api.md

```diff
@@ -25,7 +29,7 @@ Similarily, you can also use the `connect_to_server_sync` function to connect to
 
 
 <!-- tabs:start -->
-#### ** Asynchronous Client **
+#### Asynchronous Client
```

Comment:
Not sure exactly but the stars maybe needed for rendering as tabs in the ImJoy-docs.